ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -159,6 +155,87 @@ open class Tooltip: NSObject {
         UIDevice.current.endGeneratingDeviceOrientationNotifications()
     }
 
+    required public init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    @objc(MSFTooltipArrowDirection)
+    public enum ArrowDirection: Int {
+        case up, down, left, right
+
+        var isVertical: Bool {
+            switch self {
+            case .up, .down:
+                return true
+            case .left, .right:
+                return false
+            }
+        }
+
+        var opposite: ArrowDirection {
+            switch self {
+            case .up:
+                return .down
+            case .down:
+                return .up
+            case .left:
+                return .right
+            case .right:
+                return .left
+            }
+        }
+    }
+
+    @objc(MSFTooltipDismissMode)
+    public enum DismissMode: Int {
+        case tapAnywhere
+        case tapOnTooltip
+        case tapOnTooltipOrAnchor
+    }
+
+    @available(*, deprecated, message: "Screen margins value has been tokenized. Setting this var no longer has any effect and it will be removed in a future update.")
```

Comment:
give info what api they should be using.

Reply:
there's no other api to be using. the screen margins is built into the token system now